### PR TITLE
Fixed tests check_options and updated README

### PR DESCRIPTION
**Pull request by sworleys**
Fixed check_options tests and updated README. In reference to #71.

### CODE REVIEW

File: tests/check_options.py [hunk 1]
```diff
@@ -1,149 +1,259 @@
...
+    version_output, err = version.communicate()
+    print version_output
+    version_regex = re.compile(r"[0-9]\.[0-9]\.[0-9]")
+    if re.search(version_regex, version_output) is not None:
+        pass
+    else:
+        print err
+        pytest.fail("Did not print version number")
+
+def test_update_collection_rules():
```
Comment (brantleyr) [inline tests/check_options.py:299]: I would probably use some sort of creation & modification times for this for the rules files but I guess this works.

Comment (brantleyr) [inline tests/check_options.py:299]: 226-234

File: insights_client/constants.py [hunk 2]
```diff
@@ -12,23 +12,25 @@ class InsightsConstants(object):
...
-    default_conf_dir = '/etc/' + app_name + '/'
-    log_dir = '/var/log/' + app_name
-    default_log_file = log_dir + '/' + app_name + '.log'
-    default_conf_file = default_conf_dir + app_name + '.conf'
-    default_sed_file = default_conf_dir + '.exp.sed'
-    default_ca_file = default_conf_dir + 'cert-api.access.redhat.com.pem'
+    user_agent = os.path.join(app_name, 'version')
+    default_conf_dir = os.path.join(os.sep, 'etc', app_name)
+    log_dir = os.path.join(os.sep, 'var', 'log', app_name)
+    default_log_file = os.path.join(log_dir, app_name) + '.log'
```
Comment (brantleyr) [inline insights_client/constants.py]: Why do we use "+" concatenation here and os.path.join() in other places? Maybe just a styling preference and not really much functionality difference.

**Comment by gravitypriest** [comment]
please pep8 @Prosper95

**Pull request merged**